package.json

```diff
@@ -36,7 +38,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "ghbuild": "CI=false react-scripts build",
+    "ghbuild": "CI=false craco build",
```

Comment:
Are you sure we are not losing any needed data, like when we compress an image?

Reply:
Nop no data loss just better compiller